src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -160,24 +165,19 @@ private GasDistribution DistributeGas(ApplicationEngine engine, UInt160 account,
             };
         }
 
-        private BigInteger CalculateBonus(DataCache snapshot, ECPoint vote, BigInteger value, uint start, uint end)
+        private BigInteger CalculateBonus(DataCache snapshot, NeoAccountState state, uint end)
```

Comment:
`end` need to be removed, because vote reward calculation only support current height.